integration/raft/migration_test.go

```diff
@@ -201,11 +189,11 @@ var _ = Describe("Kafka2RaftMigration", func() {
 
 			By("3) Verify: create new channel, executing transaction")
 			network.CreateAndJoinChannel(orderer, channel2)
-			nwo.EnableCapabilities(network, channel2, "Application", "V2_0", orderer, network.Peer("Org1", "peer0"), network.Peer("Org2", "peer0"))
-			nwo.DeployChaincode(network, channel2, orderer, chaincode, network.PeersWithChannel(channel2)...)
-			RunExpectQueryRetry(network, peer, channel2, 100)
-			RunExpectQueryInvokeQuery(network, orderer, peer, channel2, 100)
-			RunExpectQueryInvokeQuery(network, orderer, peer, channel2, 90)
+
+			env = CreateBroadcastEnvelope(network, peer, channel2, []byte("hola"))
```

Comment:
Doing an [nwo.Broadcast ](https://github.com/hyperledger/fabric/blob/master/integration/nwo/orderer_client.go#L22-L51)doesn't mean much, as it doesn't check that the transaction was included into some block.

Perhaps:

1. Add usage of a Deliver call to ensure a new block with the expected seq num has been created
2. Extract this repetitive pattern (send transaction and ensure block is created) into a function

Reply:
+1 Performing the Deliver check would improve the quality of this test dramatically.